boardless_outside/lib/firebase_options.dart

```diff
@@ -0,0 +1,73 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+/// Default [FirebaseOptions] for use with your Firebase apps.
+///
+/// Example:
+/// ```dart
+/// import 'firebase_options.dart';
+/// // ...
+/// await Firebase.initializeApp(
+///   options: DefaultFirebaseOptions.currentPlatform,
+/// );
+/// ```
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      return web;
+    }
+    // ignore: missing_enum_constant_in_switch
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        return macos;
+    }
+
+    throw UnsupportedError(
+      'DefaultFirebaseOptions are not supported for this platform.',
```

Comment:
Firebase doesn't seem to support other platforms, should I delete files for building on Windows and Linux?

Reply:
Folders removed